src/main/java/com/iota/iri/network/Node.java

```diff
@@ -221,7 +222,7 @@ private Runnable spawnNeighborDNSRefresherThread() {
     }
 
     public void preProcessReceivedData(byte[] receivedData, SocketAddress senderAddress, String uriScheme) {
-        TransactionViewModel receivedTransactionViewModel = null;
+        TransactionViewModel receivedTransactionViewModel;
```

Comment:
I actually like explicitly setting variables null...

If we want to improve the code on PRs (which I've been told to refrain from doing) this would be one thing I wouldn't be doing.

Reply:
fair enough,
it was intellij's idea :)